src/main/java/com/zendesk/maxwell/producer/HttpPostProducer.java

```diff
@@ -0,0 +1,115 @@
+package com.zendesk.maxwell.producer;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpResponseException;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.HttpHeaders;
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpBackOffUnsuccessfulResponseHandler;
+import com.google.api.client.util.ExponentialBackOff;
+
+import com.google.api.client.http.javanet.NetHttpTransport;
+import org.apache.commons.codec.binary.Hex;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import java.io.IOException;
+
+public class HttpPostProducer extends AbstractProducer {
+    static final String HTTP_HMAC_HEADER = "x-signature";
+    static final String HTTP_CONENT_TYPE_HEADER = "application/json";
+    static final String HMAC_ALG = "HmacSHA1";
```

Comment:
the hmac code seems awfully specific to your use case.  Is there any way to make it somehow generic?  i poked briefly around the web to see if anyone had done a signed-request standard, but not how I could see.